Homework4/src/ro/siit/homework4/Store.java

```diff
@@ -0,0 +1,73 @@
+package ro.siit.homework4;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+public class Store {
+
+    static Map<String, StockItem> stock;
+    static List<SoldItem> sold;
+
+    @Override
+    public String toString() {
+        return "Store{" +
+                "stock=" + stock +
+                ", sold=" + sold +
+                '}';
+    }
+
+    public Store() {
+        this.stock = new HashMap<>();
+        this.sold = new ArrayList<>();
+    }
+
+    public static void add(StockItem product) {
+
+        stock.forEach((k, v) -> {
```

Comment:
stock is a map... you chose to put this in the map
ID: 1 value: StockItem(george)

when you add or romove you give the StockItem and search by name

why don't you make the ID the name instead of the useless number? then you could do:
stock.get(product.getProduct().getName()) /// this would be the exact stock item to add the quantity to.